tests/Fixer/Comment/CommentToPhpdocFixerTest.php

```diff
@@ -246,6 +248,70 @@ public function provideTestCases()
 EOT
                 ,
             ],
+            [
+                <<<'EOT'
+<?php /* header comment */ $foo = true;
+
+// @todo do something later
+$foo = 1;
```

Comment:
Please add a test case with a comment before a structural element, I think the added tests here would still pass without the `ignored_tags`, thus they don't prove the option works as expected.

Reply:
What do you mean? Isn't this case with a comment before a structural element? Can you prepare the one you are thinking about?

Reply:
Actually I misunderstood the code, sorry.